pkg/discovery/endpoint/client/client.go

```diff
@@ -500,10 +557,38 @@ func (w *webVDR) Read(didID string, opts ...vdrapi.DIDMethodOption) (*did.DocRes
 	return w.VDR.Read(didID, append(opts, vdrapi.WithOption(web.HTTPClientOpt, w.http))...)
 }
 
-type casReader struct {
+type defaultCASReader struct {
 	s *Client
 }
 
-func (c *casReader) Read(key string) ([]byte, error) {
-	return c.s.send(nil, http.MethodGet, fmt.Sprintf("%s/%s/%s", ipfsGlobal, "ipfs", key))
+func (c *defaultCASReader) Read(cidWithPossibleHint string) ([]byte, error) {
```

Comment:
@DRK3  @fqutishat sample default client implementation of 'combined' read (ipfs and webcas) without storage. There is TODO: for adding support for default webcas reader (without storage). There are two options: 
1. provide two separate ipfs and webcas readers in orb code base and then the client code will pick and choose what to implement/enable. 
2. provide combined reader for both ipfs and webcas in orb code base and have options so client can turn on/off ipfs (loca/global) and webcas.

Reply:
I was going to go with the first approach, @fqutishat Let me know if you feel differently